src/app/ubs-admin/components/ubs-sidebar/ubs-sidebar.component.html

```diff
@@ -1,24 +1,10 @@
-<!-- for Header -->
-<div class="wrapper">
-  <div class="logo">
-    <img src="assets/admin-header/logo-admin.svg" />
-  </div>
-  <div class="icons">
-    <img src="assets/admin-header/first-icon-admin.svg" />
-    <img src="assets/admin-header/bell-icon-admin.svg" />
-    <span>En</span>
-    <img src="assets/img/arrow_down.svg" />
-    <a href="/">Home</a>
-    <span [ngStyle]="{ color: 'var(--primary-green)', fontFamily: 'var(--primary-font)' }">Jana Done</span>
-  </div>
-</div>
 <div class="sidebar-icons" #sideBarIcons>
   <ul class="sidebar-list">
     <li class="sidebar-list-item" (click)="toggleSideBar()" *ngFor="let listItem of listElements">
       <img class="sidebar-list-item-icon" src="{{ listItem.link }}" />
     </li>
   </ul>
-  <button class="sidebar-toggler" type="button" (click)="toggleSideBar()" #sidebarToggler [disabled]="stopClick">>></button>
+  <button class="sidebar-toggler" (click)="toggleSideBar()" #sidebarToggler [disabled]="stopClick"></button>
```

Comment:
this doesn't work as expected. Can we please make some changes:
- let's move sidebar button (see screen below), so we can collapse it on clicking button instead of having all sidebar as button
<img width="651" alt="Screenshot 2021-06-05 at 00 14 21" src="https://user-images.githubusercontent.com/46226026/120863426-32ea1c80-c593-11eb-911d-ed74d6981afa.png">

Reply:
*the issue now is that we can click anywhere on sidebar and it will collapse
so please, let's make changes described above

Reply:
I moved button to the right corner